app/src/store/playground/playgroundStore.tsx

```diff
@@ -228,6 +257,9 @@ export const createPlaygroundStore = (initialProps: InitialPlaygroundState) => {
         if (savedProviderConfig != null) {
           newModel = {
             ...savedProviderConfig,
+            // we don't want to use persisted supportedInvocationParameters
+            supportedInvocationParameters:
+              currentModel.supportedInvocationParameters,
```

Comment:
we probably just shouldnt' save these anyways. I can update that in the other issue #5511